contexts/credentials/v2

```diff
@@ -112,9 +112,9 @@
       }
     },
 
-    "JsonSchema2023": {
+    "JsonSchema": {
       "@id":
-        "https://w3.org/2018/credentials#JsonSchema2023",
+        "https://www.w3.org/TR/vc-json-schema/#JsonSchema",
```

Comment:
```suggestion
        "https://w3.org/2018/credentials#JsonSchema",
```

Fixed to align with this:

https://github.com/w3c/vc-data-model/pull/1218/files